src/Windows/Avalonia.Direct2D1/Media/GeometryImpl.cs

```diff
@@ -26,8 +26,15 @@ public GeometryImpl(Geometry geometry)
         /// <inheritdoc/>
         public Rect GetRenderBounds(Avalonia.Media.Pen pen)
         {
-            var factory = AvaloniaLocator.Current.GetService<Factory>();
-            return Geometry.GetWidenedBounds((float)pen.Thickness).ToAvalonia();
+            //var factory = AvaloniaLocator.Current.GetService<Factory>();
+            if (pen == null)
+            {
+                return Geometry.GetWidenedBounds(0).ToAvalonia();
+            }
+            else
+            {
+                return Geometry.GetWidenedBounds((float)pen.Thickness).ToAvalonia();
```

Comment:
Couldn't this block be written more succinctly as: `return Geometry.GetWidenedBounds((float)(pen?.Thickness ?? 0)).ToAvalonia();`?

Reply:
I usually do that wasn't sure about readability and code style. Will have that in mind for the future.